frontend/javascripts/admin/organization/upgrade_plan_modal.tsx

```diff
@@ -299,6 +299,7 @@ function UpgradePricingPlanModal({
           ) : null}
         </>
       }
+      zIndex={10000} // overlay everything
```

Comment:
@philippotto I tried to click somewhere in the background (`document.body.click()`) to make the menu disappear but that did not work. With the high zIndex you can at least be sure that it overlays anything.